migrations/2023-07-07-091223_create_captures_table/up.sql

```diff
@@ -0,0 +1,25 @@
+
+CREATE TYPE "CaptureStatus" AS ENUM (
+    'started',
+    'charged',
+    'pending',
+    'failure'
+);
+CREATE TABLE captures(
+    id SERIAL PRIMARY KEY,
```

Comment:
I don't think we use auto generated id's. it will be difficult to move the data in future.

Reply:
All the other tables in DB has `id` field which is auto increment. 
I just did the same here.